src/auth/jwtclient.ts

```diff
@@ -68,7 +71,9 @@ export class JWT extends OAuth2Client implements IdTokenProvider {
     key?: string,
     scopes?: string | string[],
     subject?: string,
-    keyId?: string
+    keyId?: string,
+    useJWTAccessAlways?: boolean,
```

Comment:
Let's just stop adding new variables to the constructor, and if they want this, force folks to use the object approach.  This was largely here for back-compat.